apps/mobile/src/features/send/components/recipient/build-recipient-suggestions.spec.ts

```diff
@@ -12,7 +12,7 @@ import { btcAsset } from '@leather.io/constants';
 import { SendAssetActivity } from '@leather.io/models';
 
 // One of the imports references @lingui/macro, causing compile-time errors.
```

Comment:
Not a big deal but maybe we can update / remove the comment above now as it's out-dated
`One of the imports references @lingui/macro...` 

Could be `One of the imports references @lingui/core` if it is still needed

Reply:
Oh good catch, this is no longer necessary. Removed.